app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from .players.elves.elf_ranger import ElfRanger
+from .players.elves.druid import Druid
+from .players.dwarves.dwarf_warrior import DwarfWarrior
+from .players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+
+
+usage = [DwarfBlacksmith, DwarfWarrior, Druid, ElfRanger]
```

Comment:
Why did you make this variable?

Reply:
cause flake8 angry at if i not use what i'm imported, but in the task i didn't need to use it